libcudacxx/codegen/codegen.cpp

```diff
@@ -326,30 +334,52 @@ _LIBCUDACXX_BEGIN_NAMESPACE_STD
             }
             for (auto& cv : cv_qualifier)
             {
+              out << "template<class _Type, _CUDA_VSTD::__enable_if_t<sizeof(_Type)==" << sz / 8;
+              if (type.first == "f")
+              {
+                out << " && _CUDA_VSTD::is_floating_point<_Type>::value, int> = 0>\n";
```

Comment:
We should replace all the `is_meow<...>::value` with `_CCCL_TRAIT(is_meow, ...)` that way in c++14 and on we will use the cheap inline variable and not the struct

Reply:
Thanks for the fix on top of my 'fix'. Hah